packages/sdk-v2/src/sdk/claim/react.ts

```diff
@@ -101,11 +110,20 @@ export const useMultiClaim = (poolsDetails: PoolDetails[] | undefined) => {
 
   // once the next contract is set (status === "None"), perform claim
   useEffect(() => {
-    if (state.status !== "None" || !contract) return;
+    // there can be a delay between sending claim request to wallet and balance update being triggered.
```

Comment:
but you now check balance inside the effect, so why is this change necessaray?

Reply:
I need to update the copy as that is not true anymore (we don't have a dependency for balance)

I chose to keep it to be sure, as the state.status updates are bit wonky/unreliable because we overwrite the contract during the cycle.

Reply:
it doesnt make sense, you simply check the same thing twice. please update copy and logic

Reply:
what is the logic update? remove it?

Reply:
nvm, removed it.